app/javascript/controllers/booking_form_price_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="booking-form-price"
+export default class extends Controller {
+  static targets = ["startDate", "endDate", "price", "field", "days"]
+
+  enabledEndDateField() {
+    this.endDateTarget.value = ""
+    this.endDateTarget.min = new Date(new Date(this.startDateTarget.value).valueOf() + 86400000).toISOString().split('T')[0]
```

Comment:
Not that is wrong, just for curiosity, Why are you using miliseconds and not days?

Reply:
There's something called Unix Time.
Unix time is the amount of miliseconds since the 1st of January 1970 and that's the reference JS use for its Date model.
So today is `1677279600` for example.

Because of that when you want to add some time to a Date in JS you'll often use a value in millisecond so that it's added to this Unix Time.

There are other ways but this one is the shortest as far as I know